install_scripts/install_packages.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Install apt packages
+
+# First, get a fresh state
+apt update -y
+apt full-upgrade -y
+apt autoremove -y
+
+# Note: should we just apt all these for install?
+apt-get install imagemagick  	# Needed for sindri (I think)
+apt-get install eject        	# Automount drives
+apt-get install udisks2			# Automount drives
+
+# These should already be installed on the image...
+apt-get install python3-venv
+apt install git
+apt install build-essential python3-dev gfortran
+apt install networkd-dispatcher
+apt install modemmanager
```

Comment:
There's an inexplicable mix of using the `apt` and `apt-get` entrypoints here, without obvious reason. For interactive use, you should use `apt`, while for a script like this, you should use `apt-get` (since the latter interface is designed to be stable for scripts, while the former can break unexpectedly). At the very least, one should be consistent within the same script...

```suggestion
apt-get update -y
apt-get dist-upgrade -y
apt-get autoremove -y

# Note: should we just apt all these for install?
apt-get install imagemagick  	# Needed for sindri (I think)
apt-get install eject        	# Automount drives
apt-get install udisks2			# Automount drives

# These should already be installed on the image...
apt-get install python3-venv
apt-get install git
apt-get install build-essential python3-dev gfortran
apt-get install networkd-dispatcher
apt-get install modemmanager
```